app/assets/javascripts/admin/spree/orders/variant_autocomplete.js.erb

```diff
@@ -54,18 +54,21 @@ $(document).ready(function() {
     }
     $('a.save-item').click(handle_save_click);
 
-    handle_delete_click = function(e, confirmed){
-      if (confirmed) {
-        var del = $(this);
-        var shipment_number = del.data('shipment-number');
-        var variant_id = del.data('variant-id');
+    handle_delete_click = function(e){
+      var del = $(this).parent().parent().parent();
```

Comment:
Could we find this element in a different way? With this code, if the structure of the HTML changes, this function could break.